src/static/js/config/aria-states-config.js

```diff
@@ -0,0 +1,79 @@
+/* ==========================================================================
+   Aria State
+
+   Code copied from the following with minor modifications:
+
+   - https://github.com/IBM-Watson/a11y.js
+     Copyright (c) 2014 IBM
+
+   - http://www.w3.org/TR/wai-aria/states_and_properties
+   ========================================================================== */
+'use strict';
+
+var _undefined;
```

Comment:
Only tweak might be to treat it like a const value stylistically, so `var UNDEFINED`. In future-land it would be updated to `const UNDEFINED` per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const